platforms/r3-corda-ent/charts/zone/templates/_helpers.tpl

```diff
@@ -0,0 +1,5 @@
+{{- define "labels.custom" -}}
```

Comment:
labels.custom has not been used. Ideally use this in all templates

Reply:
fixed